src/components/HourlyCityWeather/HourlyCityWeather.js

```diff
@@ -0,0 +1,168 @@
+import { useEffect, useState } from "react";
+import { Col, Row } from "react-bootstrap";
+import Carousel from 'react-bootstrap/Carousel';
+import './HourlyCityWeather.css';
+
+const HourlyCityWeather = ({city}) => {
+    const [error, setError] = useState(null);
+    const [isLoaded, setIsLoaded] = useState(false);
+    const [timeStamps, setTimeStamps] = useState(0);
+
+
+    useEffect(() => {
+        fetch(
+            "https://api.openweathermap.org/data/2.5/forecast?q=" +
+            city +
+            "&units=metric" +
+            "&appid=" +
+            process.env.REACT_APP_APIKEY
+        )
+            .then((res) => res.json())
+            .then(
+                (result) => {
+                    if (result["cod"] !== "200") {
```

Comment:
Usually, bracket notation (`obj['property']`) isn't used unless we are looking up a variable property or the key property contains characters that can't be used in dot notation (`obj.property`).